### PR TITLE
feat: Creating layout for ammeter 

### PR DESCRIPTION
**Pull request by riyasingh1004**
Fixes :First steps for issues #1426 

**Changes**: 

Creating the layout for the new device ammeter

**Screenshot/s for the changes**: 

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [ ] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing**:

[device_ammeter_layout.zip](https://github.com/fossasia/pslab-android/files/2948362/device_ammeter_layout.zip)

### CODE REVIEW

**Comment by cweitat** [comment]
@riyasingh1004 please check travis errors

**Comment by mariobehling** [comment]
Travis is failing. Please check.

File: app/src/main/res/layout/activity_ammeter [hunk 1]
```diff
@@ -0,0 +1,120 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (CloudyPadmal) [inline app/src/main/res/layout/activity_ammeter]: The file name must end with .xml

**Comment by ho-dor** [comment]
Can I take this up . There has been no update for some time .

**Comment by CloudyPadmal** [comment]
@ho-dor let's give @riyasingh1004 some time to respond.

**Comment by ho-dor** [comment]
@CloudyPadmal  can i take it up now . There has been no response for 20 days now. Thanks.

**Comment by ho-dor** [comment]
@CloudyPadmal @cweitat can I take it up if this is important or else we can close this PR.

**Comment by CloudyPadmal** [comment]
@ho-dor you can continue with this issue if you like as it been sometime. Thanks.

**Comment by ho-dor** [comment]
@CloudyPadmal sure , I will create a PR ASAP.

**Pull request closed without merging**